interface-definitions/dns-forwarding.xml.in

```diff
@@ -36,6 +36,18 @@
                   <multi/>
                 </properties>
               </leafNode>
+              <leafNode name="dns64-prefix">
+                <properties>
+                  <help>Help to communicate between IPv6-only client and IPv4-only server</help>
+                  <valueHelp>
+                    <format>ipv6net</format>
+                    <description>IPv6 address and prefix length</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="ip-prefix"/>
```

Comment:
I guess it should be `ipv6-prefix`?

Reply:
> I guess it should be `ipv6-prefix`?

Exactly!
Fixed

Reply:
PDNS man page states: `Enable DNS64 (RFC 6147) support using the supplied /96 IPv6 prefix`. We probably should use this help string. Also what happens if a /64 is supplied? Does PDNS terminate? If this is the case, please add a `verify()` step to check prefix length.

Reply:
With /64 it treminated `code=exited, status=1/FAILURE`
I'll add checks

Reply:
Please also add Smoketests verifying these checks.

To test if an exception is raised use:
```python
with self.assertRaises(ConfigSessionError):
    self.cli_commit()
```

Reply:
> Please also add Smoketests verifying these checks.

Done!